package.json

```diff
@@ -81,6 +81,7 @@
     "typescript": "^5.2.2"
   },
   "dependencies": {
+    "@adobe/css-tools": "^4.3.2",
```

Comment:
look like you can remove immer

Reply:
removed

Reply:
oops, we use its functionality in `topics`